flaskApi/parser.py

```diff
@@ -0,0 +1,64 @@
+from pptx import Presentation
+
+
+def get_presentation_as_list_of_slides(path_to_presentation):
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be